docs/api-reference/3d-tiles/tileset-3d.md

```diff
@@ -142,7 +123,7 @@ street level views.
 
 Valid values are between 0.0 and 1.0.
 
-# loadProgress : Event
+# onTileLoad
```

Comment:
This should be a H3?

Reply:
Yes, this doc needs a lot of love, will fix separately.